tiled/server/compression.py

```diff
@@ -92,13 +94,17 @@ async def send_compressed(self, message: Message) -> None:
                     )  # higher is better
                     THRESHOLD = 1 / 0.9
                     if compression_ratio > THRESHOLD:
-                        headers[
-                            "X-Compression-Stats"
-                        ] = f"time={1000 * compression_time:.2f},ratio={compression_ratio:.2f}"
                         headers["Content-Encoding"] = self.encoding
                         headers["Content-Length"] = str(len(compressed_body))
                         headers.add_vary_header("Accept-Encoding")
                         message["body"] = compressed_body
+                        # The Server-Timing middleware, which runs after this
```

Comment:
is this change related to caching? Or is it mostly a related improvement to instrumentation?

Reply:
Yeah, just a related improvement to instrumentation.